scenes/game/remote_game.gd

```diff
@@ -221,7 +221,7 @@ func process_discard_to_max(action_message) -> void:
 	var card_ids = action_message['card_ids']
 	local_game.do_discard_to_max(game_player, card_ids)
 
-func do_card_from_hand_to_gauge(player : LocalGame.Player, card_ids : Array) -> bool:
+func do_relocate_card_from_hand(player : LocalGame.Player, card_ids : Array) -> bool:
 	var action_message = {
 		'action_type': 'action_card_from_hand_to_gauge',
```

Comment:
Ideally we would update this too. But do we care about replay compatibility?
I'd say probably not at this point?

Actually, isn't this how these functions are called? You renaming that is going to break it.